developers/creating-your-first-plugin.rst

```diff
@@ -192,3 +192,80 @@ Velocity automatically registers your plugin main class as a listener.
             // For instance, we could register an event:
             server.getEventManager().register(this, new PluginListener());
         }
+
+Alternatively: Kotlin
+^^^^^^^^^^^^^^^^^^^^^
+
+Kotlin is the second-most popular language capable of running on the JVM (behind Java).
+As such, it's a viable option for developing Velocity plugins; additionally, its more
+modern syntax makes it a very attractive choice.
+
+Here are some brief notes to assist in starting a Velocity plugin using Kotlin:
+
+Your build script will look a little different than the above, because of the additional
+dependency on the Kotlin stdlib and a specialized annotation processor that must be used
+in place of Gradle's default. Also, it will probably be written in Kotlin itself, rather
+than Groovy.
+
+Your ``build.gradle.kts`` should include the following:
+
+.. code-block:: kotlin
+
+    plugins {
+        kotlin("jvm") version "1.4.10"
```

Comment:
Kotlin 1.4 deprecates this plugin; use MPP in the example.

Reply:
I couldn't find any info about the deprecation you're referring to, but since this is targeting the JVM I would expect to need the JVM plugin? I tried using `kotlin("multiplatform")` instead but the build failed.